sentry-rails/lib/sentry/rails/error_subscriber.rb

```diff
@@ -27,7 +27,16 @@ def report(error, handled:, severity:, context:, source: nil)
           hint.merge!(context.delete(:hint))
         end
 
-        Sentry::Rails.capture_exception(error, level: severity, contexts: { "rails.error" => context }, tags: tags, hint: hint)
+        options = { level: severity, contexts: { "rails.error" => context }, tags: tags, hint: hint }
+
+        case error
+        when String
+          Sentry::Rails.capture_message(error, **options)
+        when Exception
+          Sentry::Rails.capture_exception(error, **options)
+        else
+          raise ArgumentError, "Expected an Exception or a String, got: #{error.inspect}"
```

Comment:
we should not raise inside the SDK, let's use `log_debug` here

Reply:
@sl0thentr0py hmm, what's the best way of accessing logger though? I can see that the helper is defined in the LoggingHelper, but not sure which object I should use for logging, or if I should include that helper in the ErrorSubscriber?

Reply:
yes you can just include it

Reply:
@sl0thentr0py sorry, but could you tell me how I should configure the logger so that I could do something like this in the spec:

```ruby
expect {
          Rails.error.report(312, severity: :info, context: { foo: "bar" })
        }.to output(/Expected an Exception or a String, got: 312/).to_stdout
```

? Including the helper is not enough as it expects `@logger` which is not set on the ErrorReporter.

Reply:
yes for that you need the instance var, but you can also use `Sentry.configuration.logger` directly